### PR TITLE
Fix memory policy default size

### PR DESCRIPTION
**Pull request by pavlospt**
I added a condition to return `maxSize=1` only if no `maxSize` set.

### CODE REVIEW

**Comment by pavlospt** [comment]
Any updates on this 😃 ?

File: build.gradle [hunk 1]
```diff
@@ -22,7 +22,7 @@ buildscript {
     ]
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath 'com.android.tools.build:gradle:3.1.0'
```
Comment (tiwiz) [inline build.gradle]: Could we move it to 3.1.2, please?

**Review by tiwiz — CHANGES_REQUESTED** [review]

**Comment by pavlospt** [comment]
@tiwiz I bumped the AGP version. Sorry for taking so long. Somehow I missed your comment's notification! :/

**Comment by tiwiz** [comment]
@pavlospt thank you! Let's wait for one more reviewer to approve and then we can merge it! Thanks!

**Review by digitalbuddha — APPROVED** [review]

**Review by tiwiz — APPROVED** [review]

**Pull request merged**